DFE-Digital/early-careers-framework
# [CPDLP-904] [CPDLP-907] Seed existing ECF standard schedules

## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDLP-904
Ticket: https://dfedigital.atlassian.net/browse/CPDLP-907

- Fixed schedule dates data for ECF Standard Schedules
- Original ECF schedules now seeded via CSV mechanism
- This means these schedules can now be load in easily in all environments when needed
- CSV seeder changed slightly so it will now update dates to whatever is present in the CSV
- Old hardcoded ECF seeds have been removed as these are now seeded via CSV
- I will call new seeds on each environment as I deploy

## Tech review

### Is there anything that the code reviewer should know?

### Code quality checks

### HTML Checks

### Gotchas

## Product review

### How can someone see it it review app?

- https://ecf-review-pr-1647.london.cloudapps.digital/finance
- Login as a finance user
- View schedules
- Should see original ECF standard schedules
- Should show correct start dates

## External API changes

- Some schedules will change data but none of these are documented any where
